src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IParameterReferenceExpression.cs

```diff
@@ -848,7 +848,7 @@ public void M(int x)
     Clauses:
         IPatternCaseClauseOperation (Label Id: 0) (CaseKind.Pattern) (OperationKind.CaseClause, Type: null) (Syntax: 'case var y  ...  (x >= 10):')
           Pattern: 
-            IDeclarationPatternOperation (Declared Symbol: System.Int32 y) (OperationKind.DeclarationPattern, Type: null) (Syntax: 'var y')
+            IDeclarationPatternOperation (Declared Symbol: System.Int32 y, AcceptsNull: True) (OperationKind.DeclarationPattern, Type: null) (Syntax: 'var y')
```

Comment:
My initial reaction to this is mild confusion, as we know that there's a guard here and that the pattern won't actually accept `null`, but on reflection I think I'm ok with it, as the user could have typed `case var y when (x == null || x > 10)`.